assets/ca/ca.querytranslator.js

```diff
@@ -263,14 +277,9 @@ var caUI = caUI || {};
 	 * @param {String} wildcardSuffix
 	 */
 	assignOperatorAndValue = function (rule, queryValue, negation, wildcardPrefix, wildcardSuffix) {
-		var matches;
 		// Determine the operator that matches the given query, negation and wildcard positions.
-		if (queryValue === '*') {
+		if (!queryValue) {
```

Comment:
The `queryValue` is never going to be `*`, that will be taken by the `wildcardPrefix`.  I'm not sure if/how this ever worked, but it does now.